tools/capture-test.py

```diff
@@ -114,6 +116,16 @@ def pcm_init_stream(pcm, rate, chans, fmt, access):
     alsa.snd_pcm_hw_params_set_channels(pcm, hwp, chans)
     alsa.snd_pcm_hw_params_set_rate(pcm, hwp, rate, 0)
     alsa.snd_pcm_hw_params_set_access(pcm, hwp, access)
+    alsa.snd_pcm_hw_params_set_buffer_size(pcm, hwp, opts.bufsz)
+    if opts.verbose:
+        print("Set hw_params:")
```

Comment:
I know you will likely reply "overkill!" but this really smells like a missed `import logging` opportunity. People often overstate the complexity of Python's built-in `logging`, for a counter example see how little code `logging` required in recent commit 5419b42ff162a75dec0a4230b5d6012b23cb39b9

Also, something resistant to change something :-D

Reply:
Logging puts ugly prefix nonsense on the output lines, which IMHO is exactly what you don't want for human readable --verbose output.  For a running system with multiple levels and the expectation of an engineer pouring through logs to intuit state?   Sure.   Here, for a --verbose flag?  That's just not the way unix tools are expected to work.

Reply:
> Logging puts ugly prefix nonsense on the output lines

No, it does not. As seen in 5419b42ff162a75dec0a4230b5d6012b23cb39b9

```python
ipython3

import logging
logging.basicConfig(level=logging.INFO, format="%(message)s")
logging.info("foo")

foo # no prefix!

```

And that's all it takes.

> Here, for a --verbose flag?

Python is a high level language. This means complicated things are possible, but simple things are easy: it's not because `logging` has 5 levels that you have to use them. You can just use two: verbose and not. Same for prefixes, multiple loggers and what not: they can be completely ignored.

What `logging` gives even when not using _any_ complex/fancy feature:
- the code is not littered with `if opt.verbose` conditionals and indentation
- best flexibility to change or add extra levels temporarily for debugging purposes with single-line changes and no wrangling with `if opt.verbose` conditionals and indentation.

Don't get me wrong: I do NOT think this PR should be blocked on switching to `logging`. I only want to thank you for confirming that `logging`'s complexity is generally  overstated and misunderstood :-)

Maybe its documentation is missing the example above...